app-benchmarks/edk2-test/spec

```diff
@@ -0,0 +1,11 @@
+VER=202406
+
+SRCS__AMD64="file::rename=SctPkg::https://github.com/tianocore/edk2-test/releases/download/edk2-test-stable${VER}/UefiSctBinaryX64.zip"
+SRCS__ARM64="file::rename=SctPkg::https://github.com/tianocore/edk2-test/releases/download/edk2-test-stable${VER}/UefiSctBinaryAARCH64.tar.xz"
+SRCS__LOONGARCH64="file::rename=SctPkg::https://github.com/tianocore/edk2-test/releases/download/edk2-test-stable${VER}/UefiSctBinaryLOONGARCH64.tar.gz"
+
+CHKSUMS__AMD64="sha256::9a2e96dd4a409f39369603178837b3d130d5c173486ab011574a656368393dc3"
+CHKSUMS__ARM64="sha256::901db96eabc7a633159e8c1414343bae514657e18222ee2f0c9be2a23f8752f8"
+CHKSUMS__LOONGARCH64="sha256::07615448d9862369e9922e854dc4c82fc519ea155ae5ec567c4b12446097d836"
+SUBDIR=.
+CHKUPDATE="github::repo=tianocore/edk2-test"
```

Comment:
Please use `anitya::` where possible, this would correspond to https://release-monitoring/project/{}.